build.gradle

```diff
@@ -22,3 +22,12 @@ allprojects {
         }
     }
 }
+
+// Required for the 'publish' plugin
+subprojects {
```

Comment:
Add TODO here, `subprojects` block should not be used to declare repositories.

Reply:
removed. I added a `pluginManagement` to the main `settings.gradle`